components/LinkGrid/src/index.scss

```diff
@@ -186,6 +186,22 @@
   }
 }
 
+.denhaag-link-grid__item:hover {
+  background-color: var(--denhaag-link-grid-item-hover-background-color);
+  cursor: pointer;
+  .denhaag-link-grid__item-action {
+    transform: translateX(12px);
+  }
+}
+
+.denhaag-link-grid__item:focus-visible {
+  background-color: var(--denhaag-link-grid-item-focus-background-color);
+  border: var(--denhaag-link-grid-item-focus-border) var(--denhaag-link-grid-item-focus-border-color) !important;
```

Comment:
Die !importants kun je voorkomen door de default variabelen van `.denhaag-link-grid__item` te overschrijven binnen deze selector:
```
.denhaag-link-grid__item:focus-visible {
  --denhaag-link-grid-item-default-border: var(--denhaag-link-grid-item-focus-border);
--denhaag-link-grid-item-border-color: var(--denhaag-link-grid-item-focus-border-color);
border: var(--denhaag-link-grid-item-default-border) var(--denhaag-link-grid-item-border-color);
}
```

Reply:
Hey Robbie,

Die had ik geprobeerd maar in dit geval lost dat het niet op, omdat de border-left dan mist:
<img width="664" alt="Screenshot 2023-05-24 at 15 23 19" src="https://github.com/nl-design-system/denhaag/assets/131674048/6af79184-9c4e-4519-a5cb-a1374151dfde">

Dat komt omdat ik in de code deze CSS gebruik om die interne grid te kunnen renderen (zodat er geen dubbele border komt):

```
&:nth-child(odd) {
    border-left: var(--denhaag-link-grid-item-no-border);
}
```

Reply:
@bleddynacato check, zet maar door dan